Week3/homework/step2-2.js

```diff
@@ -2,22 +2,32 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+    if (i % 3 === 0) {
+      threeCallback();
+    }
+    if (i % 5 === 0) {
+      fiveCallback();
```

Comment:
fiveCallback & threeCallback take a number. So you should pass in the index into them.